src/main/java/software/amazon/awssdk/crt/CRT.java

```diff
@@ -152,40 +153,83 @@ public static String getCRuntime(String osIdentifier) {
             return NON_LINUX_RUNTIME_TAG;
         }
 
-        Runtime rt = Runtime.getRuntime();
-        String[] commands = {"ldd", "--version"};
-        try {
-            java.lang.Process proc = rt.exec(commands);
-
-            // the "normal" input stream of the proc is the stdout of the invoked command
-            BufferedReader stdOutput = new BufferedReader(new
-                    InputStreamReader(proc.getInputStream()));
-
-            // sometimes, ldd's output goes to stderr, so capture that too
-            BufferedReader stdError = new BufferedReader(new
-                    InputStreamReader(proc.getErrorStream()));
-
-            String line;
-            StringBuilder outputBuilder = new StringBuilder();
-            while ((line = stdOutput.readLine()) != null) {
-                outputBuilder.append(line);
+        // The system might have both musl and glibc on it:
```

Comment:
Currently, CRT java code logs via our native logger, but we can't use that yet because the C code isn't loaded...

The only place we do `System.err.println()` is when the CRT has failed to load.

I'm hesitant to print debug-but-not-error info to stderr or stdout. People hate when libraries do that.

I'm just going to 🤷‍♀️ for now